red-hat-storage/cephci
# Adding tests to verify mirroring delay in journal mirror

Automation of test case - CEPH-11491 - [Rbd Mirror] - Delayed Replication - Need to test the limit of delay that can be set. Set a reasonable high delay.

 Test Case Flow
    1. Bootstrap two CEPH clusters and setup snapshot based mirroring in between these clusters
    2. Create pools and images as specified, enable journal based mirroring for all these pools
    3. Set mirroring delay using ceph config set, verify the delay
    4. Set mirroring delay using rbd image-meta set conf_rbd_mirroring_replay_delay, verify
